oidc-api/src/test/java/uk/gov/di/authentication/oidc/services/AccountInterventionServiceTest.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.di.authentication.oidc.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.di.authentication.oidc.entity.AccountInterventionStatus;
+import uk.gov.di.orchestration.shared.services.AuditService;
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+class AccountInterventionServiceTest {
+
+    private final ConfigurationService config = mock(ConfigurationService.class);
+    private final AuditService audit = mock(AuditService.class);
+    private final HttpClient httpClient = mock(HttpClient.class);
+
+    private static String ACCOUNT_INTERVENTION_SERVICE_RESPONSE_SUSPEND_REPROVE = """
```

Comment:
As there's likely to be more of these hard-coded strings would it make sense to store them elsewhere? Or have a way of generating them from `AccountInterventionResponse`?

Reply:
🤔 Do we have a pattern for this elsewhere? Part of this was to check that the extra stuff we don't need is just ignored, so our `AccountInterventionResponse` is not capable of generating this, nor should it be. I'm relatively happy for this to be strings rather than creating bunch of code to manage it. Also it's not really needed else where. The option I'd thought about was to test that our config of the JSON parsing was doing what we wanted in a test of the `AccountInterventionResponse`, in which case we would move most of this out to there. But again I think this is reasonable.